_developer-files/connect/api/objects/form-properties/sources/saas/ms-teams-object.md

```diff
@@ -0,0 +1,64 @@
+---
+# -------------------------- #
+#     USING THIS TEMPLATE    #
+# -------------------------- #
+
+## NEED HELP USING THIS TEMPLATE? SEE:
+## https://docs-about-stitch-docs.netlify.com/reference/connect-templates/destination-form-property/
+## FOR INSTRUCTIONS & REFERENCE INFO
+
+
+# -------------------------- #
+#        CONTENT TYPE        #
+# -------------------------- #
+
+product-type: "connect"
+content-type: "api-form"
+form-type: "source"
+key: "source-form-properties-ms-teams-object"
+
+
+# -------------------------- #
+#        OBJECT INFO         #
+# -------------------------- #
+
+title: "Microsoft Teams Source Form Property"
+api-type: "platform.ms-teams"
+display-name: "Microsoft Teams"
+
+source-type: "saas"
+docs-name: "ms-teams" # This should be whatever integration.name is. Ex: LinkedIn Ads is linkedin-ads
+
+
+# -------------------------- #
+#       OAUTH PROPERTIES     #
+# -------------------------- #
+
+oauth-link: "https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-v2-protocols"
+
+oauth-description: ""
+
+oauth-attributes:
+  - name: "client_id"
+    type: "string"
+    required: true
+    credential: false
+    description: |
+      Your OAuth application's client ID, obtained when you create an app with Azure.
```

Comment:
**This** is where I'd link to the instructions in the Singer repo for setting up an Azure app.